cves/CVE-2013-0848.yml

```diff
@@ -31,7 +31,7 @@ reported_instructions: |
   you can't find it out.
 
   Please enter your date in YYYY-MM-DD format.
-reported_date:
+reported_date: 2013-12-07
```

Comment:
you sure it was reported, announced, and published on the same day? It could have been, but if there isn't concrete proof I would leave this blank